jupyter-extension/src/plugins/mount/mount.tsx

```diff
@@ -422,6 +422,9 @@ export class MountPlugin implements IMountPlugin {
     const currentMountDir = this._mountBrowser.model.path
       .split(MOUNT_BROWSER_NAME)[1]
       .split('/')[0];
+    if (currentMountDir === "out") {
```

Comment:
Yes, sorry! Basically:
* we have our poller (call verifyBrowserPath](https://github.com/pachyderm/pachyderm/blob/5cb896f6ed5fa177859d156d59f3ce68d29ecf46/jupyter-extension/src/plugins/mount/mount.tsx#L266) every time the `mountedSignal` is triggered (which I think is every 2 seconds if the connection is healthy).
* [verifyBrowserPath](https://github.com/pachyderm/pachyderm/blob/5cb896f6ed5fa177859d156d59f3ce68d29ecf46/jupyter-extension/src/plugins/mount/mount.tsx#L416-L431) resets the custom file browser to the top level if the currently open path isn't underneath any of the currently mounted repos.
* However, if the path you have open is `/pfs/out`, `verifyBrowserPath` fails the above check. So once the poller starts (which I think takes a little while sometimes), while you have `/pfs/out` open, the file browser jumps to the top level

This PR fixes the above error by short-circuiting out of `verifyBrowserPath` if the currently-mounted path is `/pfs/out` specifically.

I tested this manually; we should have an automated test for this (and I'll file an issue to track that), but I don't know how to test this currently without cypress, as this is all Lumino code, rather than React.

Reply:
Just updated the PR description as well.

Reply:
Ah ok, do you think this could be broken out into a standalone method that could be unit tested? I don't know what the `_mountBrowser` is but maybe it could be passed in? Just a thought -- I know most of this jupyter stuff is pretty hard to separate

Reply:
Yup, done. I left the `_mountBrowser` object itself out of the refactored method (and just had it pass in the active path), as that did indeed prove hard to mock, but the rest is now in  `isValidBrowserPath` and there's a test for it.